### DISCUSSION TITLE
network adapter selection

### DISCUSSION THREAD

**Discussion by metin-kahraman** [dtcooper/raspotify#608] [Q&A]
Hi ,
My raspberry has 2 connections.I'm using VPN on wired connection so i can't show the device on spotify, its okey.
My second connection is connected to local network directly via wifi but still I can't show device on spotify.
If i disabled the wired connection it works fine.
Can I choose the network adapter? (like eth0 or wlan0)
Or is there any way to fix this situation?

**Comment by JasonLG1979** [comment]
Librespot does not have that feature build in and the systemd version that Debian stable uses is not new enough to include that functionality either. Currently the best way to achieve what you're looking for is containerization.

**Reply by metin-kahraman** [reply]
Okey thank you.
